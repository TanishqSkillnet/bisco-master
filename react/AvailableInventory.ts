import AvailableInventory from './components/Inventory/AvailableInventory'

export default AvailableInventory
