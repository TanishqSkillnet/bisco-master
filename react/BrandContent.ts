import BrandContent from './components/BrandContent/BrandContent'

export default BrandContent
