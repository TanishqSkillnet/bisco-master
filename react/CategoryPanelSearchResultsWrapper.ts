import CategoryPanelSearchResultsWrapper from './components/CategoryPanel/CategoryPanelSearchResultsWrapper'

export default CategoryPanelSearchResultsWrapper