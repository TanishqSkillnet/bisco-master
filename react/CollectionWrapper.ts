import CollectionWrapper from './components/CollectionWrapper/CollectionWrapper'

export default CollectionWrapper