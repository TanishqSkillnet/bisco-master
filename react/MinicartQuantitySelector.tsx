import MinicartQuantitySelector from './components/MinicartQuantitySelector/MinicartQuantitySelector'

export default MinicartQuantitySelector
