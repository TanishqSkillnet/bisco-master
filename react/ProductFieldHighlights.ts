import ProductFieldHighlights from './components/ProductFieldHilights/ProductFieldHighlights'

export default ProductFieldHighlights