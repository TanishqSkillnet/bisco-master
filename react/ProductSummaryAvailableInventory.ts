import ProductSummaryAvailableInventory from './components/Inventory/ProductSummaryAvailableInventory'

export default ProductSummaryAvailableInventory
