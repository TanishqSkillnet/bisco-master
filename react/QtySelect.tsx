import QtySelect from './components/QuantitySelector/QtySelect'

export default QtySelect
