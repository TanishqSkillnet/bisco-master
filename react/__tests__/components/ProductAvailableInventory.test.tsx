import { render } from '@vtex/test-tools/react'
import React from 'react'
import ProductAvailableInventory from '../../components/Inventory/ProductAvailableInventory'

describe('<ProductAvailableInventory />', () => {
  const renderComponent = () => {
    return render(<ProductAvailableInventory/>)
  }

  it('should be mounted', () => {
    const { asFragment } = renderComponent()
    expect(asFragment()).toBeDefined()
  })

  it('should match the snapshot', () => {
    const { asFragment } = renderComponent()
    expect(asFragment()).toMatchSnapshot()
  })

  it('should have available inventory 100', () => {
    const { getByText } = renderComponent()
    expect(getByText('Items Available: 100')).toBeDefined()
  })
})
