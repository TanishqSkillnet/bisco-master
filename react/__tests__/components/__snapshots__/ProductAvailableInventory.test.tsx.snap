// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductAvailableInventory /> should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="availableInventoryContainer fl"
  >
    <span>
      Items Available: 100
    </span>
  </div>
</DocumentFragment>
`;
